about/about.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../about.css">
+    <title>Proyect Capstone</title>
+</head>
+
+<body>
+    
+     <div id="menu" class="">
+
+         <ul id="elements" class="">
+
+             <li>
+                <a href="#intro">ABOUT</a>
+             </li>
+             
+             <li>
+                <a href="https://goo.gl/maps/TWgiApNouQkwDre69">JOIN</a>
+             </li>
+             
+             <li>
+                <a href="#logo">LOGO</a>
+             </li>
+             
+             <li>
+                <a href="#plans">PLANS</a>
+             </li>
+             
+             <li>
+                <a href="#partners">Sponsors</a>
+             </li>
+         </ul>
+     </div>
+
+     <!--NAV TOP, NAV, -->
+     <div class="intro-background">
+        <!--NAV TOP Y NAV-->
+
+        <header>
+            <nav>
+                <div class="top">
+                    <ul>
+                        <li>
+                            <a href="https://facebook.com/"><img src="../images/logo-facebook.png" alt="facebook"></a>
+                        </li>
+                        <li>
+                            <a href="https://www.instagram.com/"><img src="../images/logo-instagram.png" alt="instagram"></a>
+                        </li>
+                        <li>
+                            <a href="">English</a>
+                        </li>
+                        <li>
+                            <a href="">My page</a>
+                        </li>
+                        <li>
+                            Partners
+                        </li>
+                    </ul>
+                </div>
+
+                <div class="bottom">
+                    <button id="hamburguer-button"></button>
+                     <ul>
+                        <li>
+                            <a href="#intro">About</a>
+                        </li>
+                        <li>
+                            <a href="https://goo.gl/maps/TWgiApNouQkwDre69">Directions</a>
+                        </li>
+                        <li>
+                            <a href="#logo">Logo</a>
+                        </li>
+                        <li>
+                            <a href="#plans">Plans</a>
+                        </li>
+                     </ul>
+                </div>
+            </nav>
+
+        </header>
+        
+         <!--description-->
+         <section id="intro">
+             
+             <ul>
+
+                 <li><a href="/index.html"><img src="../images/menu-house.png"></a></li>
+                 
+                 <li><img src="../images/arrow.png"></li>
+                
+                 <li><p>More of New Company</p></li>
+
+             </ul>
```

Comment:
- [ ] From the design given, this should be a logo that leads to the homepage of the application which is the index 👍 
- [ ] Kindly add the logo to the right with the right link as given  and p👍 

### Your view:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/825fa236-1221-43e4-ab13-818090747991)

### Design:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/0beecd45-2af7-4ab6-af2b-bdad3b82b773)